test/tests/fips_test_suite/services.rs

```diff
@@ -140,6 +298,103 @@ pub fn exec_cmd_stash_measurement<T: HwModel>(hw: &mut T) {
     assert_eq!(stash_measurement_resp.dpe_result, 0);
 }
 
+pub fn exec_fw_info<T: HwModel>(hw: &mut T) {
+    let payload = MailboxReqHeader {
+        chksum: caliptra_common::checksum::calc_checksum(u32::from(CommandId::FW_INFO), &[]),
+    };
+    let fw_info_resp = mbx_send_and_check_resp_hdr::<_, FwInfoResp>(
+        hw,
+        u32::from(CommandId::FW_INFO),
+        payload.as_bytes(),
+    )
+    .unwrap();
+
+    // Verify command-specific response data
+    assert_eq!(fw_info_resp.fmc_revision, image_revision().unwrap());
```

Comment:
I think this revision check needs to be changed to just "contains_some_data" so it's compatible with released versions as well. The structs that are supposed to handle this won't really work since there is more than one commit/revision for ROM and FW that can correspond to a released version. (HW may be more predictable?)

Reply:
I changed ROM to use "contains_some_data" but I added some logic to extract the fmc and rt revisions to pull from the fw_image, and tested that with released bins.